docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,241 @@
+  the bot can join from.
+
+  ```
+  kind: token
+  version: v2
+  metadata:
+    # The token name is not a secret because instances must prove that they are
+    # running in your AWS account to use this token.
+    name: iam-token
+    # Set a long expiry time, the default for tokens is only 30 minutes.
+    expires: "3000-01-01T00:00:00Z"
+  spec:
+    # Only allow bots to join using this token.
+    roles: [Bot]
+  
+    # Set the join method to be IAM.
+    join_method: iam
+
+    # Define the name of the bot that will be allowed to use this token. Note,
+    # the "bot-" prefix is required, the name of the bot you will create is "robot".
+    bot_name: bot-robot
+  
+    allow:
+    # Restrict the AWS account and (optionally) ARN that can use this token.
+    # This information can be obtained from running the
+    # "aws sts get-caller-identity" command from the CLI.
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+  ```
+
+  Next, create the bot user.
+
+  ```
+  $ tctl bots add robot --token=iam-token --roles=access
+  ```
+  </TabItem>
+</Tabs>
+
+## Step 3/4. Start Machine ID
+
+Now start Machine ID using the `tbot` binary. The `tbot start` command will
+start running Machine ID in a loop writing renewable certificates to
+`/var/lib/teleport` and the short lived certificates your application will use
+to `/opt/machine-id`.
+
+In a production environment you will want to run Machine ID in the background
+using a service manager like systemd. However, in this guide you will run it in
+the foreground to better understand how it works.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tbot start \
+     --data-dir=/var/lib/teleport
+     --destination-dir=/opt/machine-id \
+     --token=00000000000000000000000000000000 \
+     --join-method=token \
+     --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+     --auth-server=auth.example.com:3025
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  ```code
+  $ tbot start \
+     --data-dir=/var/lib/teleport
+     --destination-dir=/opt/machine-id \
+     --token=iam-token \
+     --join-method=iam \
+     --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+     --auth-server=auth.example.com:3025
+  ```
+  </TabItem>
+</Tabs>
+
+Replace the following fields with values from your own cluster.
+
+* `token` is the token output by `tctl bots add` command or the name of your IAM method token
+* `ca-pin` is the CA Pin for your Teleport cluster output by the `tctl bots add` command
+* `destination-dir` is where Machine ID writes the renewable certificates to, this certificate is only used by Machine ID and should not be used by applications and tools
+* `data-dir` is where Machine ID writes the short lived certificate to, this certificate should be used by applications and tools
```

Comment:
```suggestion
* `data-dir` is where Machine ID writes the short-lived certificate. This certificate should be used by applications and tools.
```

Reply:
If we accept this suggestion, we should also add a `.` to the end of the other items in this list.